script/Ciel/Day 14/14_01B_03_CIEL14B_5B.txt

```diff
@@ -1026,52 +1207,56 @@ Right now, the only thing on my mind is that light.
 [sha:9d2d991c6e12818a147128f74f217d31ea31b36a]{
 -- Page 278, Offset 41063.
 -- 引力操作で着地点を定める。
-I establish my landing spot through gravitational manipulation.
+I use Attraction to control where I'll land.
 }
 [sha:3612ea24d40dcd9d340d6fbc9bdfaa612c3d30df]{
 -- Page 278, Offset 41064.
 -- 200メートル直下の、光体の心臓に向けて落下する。
-I fall down to the heart of the Luminary 200 meters below.
+I fall straight down toward the heart of the Luminary,
+ two hundred meters below me.
 }
 [sha:b282c580b41c998ce6df951c4c03b19e2db5cd5c]{
 -- Page 279, Offset 41065.
 -- 光体からの反撃はない。
-There are no counterattacks coming from the Luminary.
+The Luminary makes no attempt to counterattack.
 }
 [sha:0840d54d6888c0007a0c83cdcb6b4f02f8c6efea]{
 -- Page 279, Offset 41066.
 -- 無防備なのも当然だ。文字通り丸裸にされたのだから。
-Of course it would be fully exposed. After all, it's been stripped completely naked.
+Of course it doesn't――it's completely defenseless.
+ I've literally stripped it naked.
 }
 [sha:14fb25f8de713451d3b80dc6dbf73f060da25f79]{
 -- Page 279, Offset 41067.
 -- とはいえ、霧散したのは中心部分だけのところを見ると、この状態も長くは持たない。
-But still, this situation will not last much longer seeing how its center is the only part that was dispersed.
+Still, when you consider that only the central column of the Luminary got vaporized,
+ it's clear this state isn't going to last.
 }
 [sha:33b085328e37a9e4901a8c016655736d1a279e22]{
 -- Page 279, Offset 41068.
 -- 光体はすぐに元に戻るだろう。
-The Luminary will surely return back to its original state soon.
+The Luminary is going to restore itself to normal before long.
 }
 [sha:ea820cbf389ca1769224f5c079633d78ab66fbc3]{
 -- Page 279, Offset 41069.
 -- その前にあの“心臓”を解体する。
-Before that happens, I'll dismantle that 'heart'.
+But before it can, I'll dismantle that 'heart'.
 }
 [sha:68b391aa18aff2570a703e1aef55eae4e6a48cba]{
 -- Page 280, Offset 41070.
 -- 「ふぅ―――――――――！」
-"Haah―――――――――!"
+"%{i}Haah%{/i}―――!"
 }
 [sha:17eca8b3556e4bf90d867d3bf212359ef1d73001]{
 -- Page 280, Offset 41071.
 -- この速度では心臓に留まる事はできない。
-I won't be able to stop at the heart at this speed.
+I'm going too fast to stop at the heart.
 }
 [sha:266ff85c45b03b9c5115c274d135fee7775a38fe]{
 -- Page 280, Offset 41072.
 -- 接触は一瞬だけ。落ちながら、すれ違う事しかできない。
-I only need to touch it once. I have no choice but to pass it as I fall.
+I only need to be in range for a split second.
+ I'll just have to strike it as I speed by.
```

Comment:
Hmmm, it's not quite what it says, but maybe it'll do? I dunno. It's a bit too 'freely written'.
Inverse the structure and it connects a bit better.
>I will pass it by as I'm falling.
>That's the best I can do.
>I will only be in range for a split second.